posthog/api/hog_function_template.py

```diff
@@ -70,6 +70,10 @@ def list(self, request: Request, *args, **kwargs):
             if sub_template_id and sub_template_id not in template.id:
                 continue
 
+            if request.path.startswith("/api/public_hog_function_templates"):
+                if "[CDP-TEST-HIDDEN]" in template.name or template.status == "alpha":
```

Comment:
The other thing is only for hog functions, not templates
```suggestion
                if template.status == "alpha":
```